### PR TITLE
make prefix dynamic and configurable

### LINKED ISSUE #30: Feat: prefix
- The prefix of the bot is _supposed to be configurable_. But, we haven't implemented it yet.
- The task here is to implement the dynamic prefixes depending upon the guild. ( [Hint ↗](https://github.com/ridays2001/Julis/blob/main/src/client/JulisClient.js#L91) )
- **WARNING!** if you implement this, the `mentionListener` would break. ( [Hint - changing this line would fix it ↗](https://github.com/the-programmers-hangout/Chronos/blob/f07f19fe7b356adcd3c4f809f0aced83694236d3/src/listeners/mentionListener.js#L28) )

### PR DESCRIPTION
**Pull request by codevenkat**
This PR fixes #30

### CODE REVIEW

**Comment by codevenkat** [comment]
Hmm `Array.find(...)` might've been a cleaner solution? Or is this fine?

**Review by ridays2001 — APPROVED** [review]
Found a mistake. Rest all looks fine to me 👍🏻

File: src/index.js [hunk 1]
```diff
@@ -16,7 +17,17 @@ class ChronosClient extends AkairoClient {
...
+      prefix: (message) => {
+        let prefix;
+        
+        configFile.guildConfigurations.forEach((conf, i) => {
+          if (conf.guildID === message.guild.id) {
+            prefix = conf.prefix;
+          }
+        });
+
+        return prefix;
```
Comment (ridays2001) [inline src/index.js]: This can be changed to `return prefix || '+';`

**Review by ridays2001 — CHANGES_REQUESTED** [review]
If there is no saved prefix for the server, the bot won't have a default prefix. Return a default prefix '+' if there are no saved prefixes.

**Review by ridays2001 — APPROVED** [review]
Okie. Should work now, I guess 👍🏻

**Review by sidtr0 — DISMISSED** [review]
It’d be nice if `configFile` could be changed to just `config` to remain consistent with other files in the project. Other than that, I don’t see any issue on first glance. I’ll get on my PC soon and will test it.

**Comment by codevenkat** [comment]
@DracTheDino there was a `config();` function in the index.js file from `require('dotenv')`, which is why I didn't name it that

**Comment by sidtr0** [comment]
> @DracTheDino there was a `config();` function in the index.js file from `require('dotenv')`, which is why I didn't name it that

Okay then, no problem. I’ll dismiss the review and then test it out on my PC. If all’s good I’ll merge it 👍

**Pull request closed without merging**